compiler/ilgen/OMRIlBuilder.hpp

```diff
@@ -539,6 +539,39 @@ class IlBuilder : public TR::IlInjector
                uint32_t numCases,
                ...);
 
+   /**
+    * @brief Generates a table switch-case control flow structure.
+    *
+    * @param selectionVar the variable to switch on.
+    * @param defaultBuilder the builder for the default case.
+    * @param geenerateBoundsCheck generate the bounds check or not for the range of the cases
```

Comment:
typo and I think I'd prefer this parameter at the end for more consistency with `Switch`

Reply:
If I move this parameter to be the last parameter in this function then it will be inconsistent with the var_args version. I am not sure how to define it in the Client API if I do that. The only support I can see for var_args in the client API is just a flag on the parameters saying that the array may also be a var_arg.

Reply:
ah, good point! I suppose we could make it "consistent" by making it the first parameter then. But I'm not sure consistency is all that valuable here. I accept the parameter where it is.